llmfoundry/models/mpt/modeling_mpt.py

```diff
@@ -636,8 +650,9 @@ def forward(self, batch):
 
     def loss(self, outputs, batch):
         targets = self.get_targets(batch)
-        return self.loss_fn(outputs.logits.view(-1, outputs.logits.size(-1)),
-                            targets.view(-1))
+        model_output = self.loss_fn(
```

Comment:
this isn't really the model output right? its the loss?

Reply:
yeah that edit wasn't needed...